rules/windows/sysmon/sysmon_mimikatz_сreds_dump.yml

```diff
@@ -0,0 +1,25 @@
+title: Mimikatz сred access dump
+description: Detects process access to LSASS which is typical for like Mimikatz tools different version
+references:
+  - http://security-research.dyndns.org/pub/slides/FIRST2017/FIRST-2017_Tom-Ueltschi_Sysmon_FINAL_notes.pdf
+tags:
+  - attack.credential_access
+  - attack.t1003
+status: experimental
+author: Aleksey Potapov, oscd.community
+date: 2019/10/23
+logsource:
+  product: windows
+  service: sysmon
+detection:
+  selection:
+    EventID: 10
+    TargetImage: 'C:\windows\system32\lsass.exe'
+    GrantedAccess:
+      - '0x1410'
+      - '0x1010'
```

Comment:
This rule was already (partially) [present](https://github.com/Neo23x0/sigma/blob/master/rules/windows/sysmon/sysmon_mimikatz_detection_lsass.yml) in Sigma repository:

```
title: Mimikatz Detection LSASS Access
status: experimental
description: Detects process access to LSASS which is typical for Mimikatz (0x1000 PROCESS_QUERY_ LIMITED_INFORMATION, 0x0400 PROCESS_QUERY_ INFORMATION "only old versions", 0x0010 PROCESS_VM_READ)
references:
    - https://onedrive.live.com/view.aspx?resid=D026B4699190F1E6!2843&ithint=file%2cpptx&app=PowerPoint&authkey=!AMvCRTKB_V1J5ow
    - https://cyberwardog.blogspot.com/2017/03/chronicles-of-threat-hunter-hunting-for_22.html
tags:
    - attack.t1003
    - attack.s0002
    - attack.credential_access
    - car.2019-04-004
logsource:
    product: windows
    service: sysmon
detection:
    selection:
        EventID: 10
        TargetImage: 'C:\windows\system32\lsass.exe'
        GrantedAccess:
            - '0x1410'      
            - '0x1010'      
    condition: selection
falsepositives:
    - unknown
level: high
```

Here is the [improvement](https://github.com/Neo23x0/sigma/blob/92e09db9ab3ea7ab94a694e220f873549117c3a4/rules/windows/sysmon/sysmon_mimikatz_detection_lsass.yml) of this rule by oscd.community:

```
title: Mimikatz Detection LSASS Access
status: experimental
description: Detects process access to LSASS which is typical for Mimikatz (0x1000 PROCESS_QUERY_ LIMITED_INFORMATION, 0x0400 PROCESS_QUERY_ INFORMATION "only old versions", 0x0010 PROCESS_VM_READ)
author: Florian Roth, Roberto Rodriguez, Dimitrios Slamaris, Mark Russinovich, Thomas Patzke, Teymur Kheirkhabarov, Sherif Eldeeb, James Dickenson, Aleksey Potapov, oscd.community (update)
date: 2017/02/16
modified: 2019/11/01
references:
    - https://onedrive.live.com/view.aspx?resid=D026B4699190F1E6!2843&ithint=file%2cpptx&app=PowerPoint&authkey=!AMvCRTKB_V1J5ow
    - https://cyberwardog.blogspot.com/2017/03/chronicles-of-threat-hunter-hunting-for_22.html
    - https://www.slideshare.net/heirhabarov/hunting-for-credentials-dumping-in-windows-environment
tags:
    - attack.t1003
    - attack.s0002
    - attack.credential_access
    - car.2019-04-004
logsource:
    product: windows
    service: sysmon
detection:
    selection:
        EventID: 10
        TargetImage: '*\lsass.exe'
        GrantedAccess|contains:
            - '0x40'
            - '0x1000'
            - '0x1400'
            - '0x100000'
            - '0x1410'    # car.2019-04-004
            - '0x1010'    # car.2019-04-004
            - '0x1438'    # car.2019-04-004
            - '0x143a'    # car.2019-04-004
            - '0x1418'    # car.2019-04-004
    filter:
        ProcessName:
            - '*\wmiprvse.exe'
            - '*\taskmgr.exe'
            - '*\procexp64.exe'
            - '*\procexp.exe'
            - '*\lsm.exe'
            - '*\csrss.exe'
            - '*\wininit.exe'
            - '*\vmtoolsd.exe'
    condition: selection and not filter
falsepositives:
    - Legitimate software accessing LSASS process for legitimate reason; update the whitelist with it
level: high
```

@alexpetrov12 I've added your authorship to it. Please consider deleting this one as redundant.